backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/picking/api/impl/PackagingDAO.java

```diff
@@ -142,7 +146,7 @@ private IQuery<I_M_Packageable_V> createQuery(@NonNull final PackageableQuery qu
 		}
 
 		//
-		// Filter by Locked By
+		// Filter by Locked By - this are not the T_Lock records, but a decidated shipment-schedule-to-user-lock 
```

Comment:
pls also add `(M_ShipmentSchedule_Lock)` just to know about which table we are talking about